# Configuration file for the microphone
speaker_param:
  chunk_size: 512
  total_channels: 1
  audio_rate: 22050
  device_name: "sysdefault"